packages/alfa-rules/src/common/predicate/is-clipped.ts

```diff
@@ -64,6 +64,52 @@ function isClippedBySize(
           }
         }
       }
+
+      if (x.value === "auto" || y.value === "auto") {
+        const { value: height } = style.computed("height");
+        const { value: width } = style.computed("width");
+
+        for (const dimension of [height, width]) {
+          switch (dimension.type) {
+            case "percentage":
+              if (dimension.value <= 0) {
+                return true;
+              } else {
+                break;
+              }
+
+            case "length":
+              if (dimension.value <= 1) {
```

Comment:
If this case, we need <=0. If the height/width is 1 pixel big, it is visible (not much, but still…)
```suggestion
              if (dimension.value <= 0) {
```